alacritty.yml

```diff
@@ -497,45 +501,79 @@
 #   - ScrollToTop
 #   - ScrollToBottom
 #   - ClearHistory
+#       Remove the terminal's scrollback history.
 #   - Hide
+#       Hide the Alacritty window.
 #   - Minimize
+#       Minimize the Alacritty window.
 #   - Quit
+#       Quit Alacritty
 #   - ToggleFullscreen
 #   - SpawnNewInstance
+#       Spawn a new instance of Alacritty that preserves the path. Only applies to Linux/BSD.
 #   - ClearLogNotice
+#       Clear Alacritty's UI warning and error notices.
```

Comment:
```suggestion
#       Clear Alacritty's UI warning and error notice.
```

While there can be multiple errors, there's only one warning/error message bar that this clears. So I think that makes it a bit more clear that we just refer to this one thing here.